<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>620</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>数据库管理工具</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/Image/app.ico</normaloff>:/Image/app.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>900</width>
     <height>620</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>900</width>
     <height>620</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget{
background-color: rgb(255, 255, 255);
}

QToolButton{
font: 24pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border:2px solid  rgb(0, 125, 165);
border-radius:15px;
background-color: rgb(0, 125, 165);
}
QLabel{
font: 14pt &quot;微软雅黑&quot;;
}
QStackedWidget{
color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
border:1px solid  rgb(0, 125, 165);
}

QTableWidget{
font: 14pt &quot;微软雅黑&quot;;
gridline-color: rgb(0, 0, 0);
border:1px solid rgb(0, 122, 165);
border-radius:5px;
background-color: rgb(255, 255, 255);
}
QTableWidgetItem{
outline:0px;
border:1px solid rgb(0, 0, 0);
}

QGroupBox{
font: 16pt &quot;微软雅黑&quot;;
color:rgb(0, 125, 165);
color:rgb(0,   0,   0);
border:1px none rgb(0, 125, 165);
border-radius:5px;
margin-top:20px;
}
QGroupBox:title{
subcontrol-origin: margin;
left: 5px;
}
QScrollBar:vertical{
width:20px;
background:rgb(0,0,0,0%);
margin:0px,0px,0px,0px;
padding-top:25px;   
padding-bottom:25px;    
}
QScrollBar::handle:vertical{
width:10px;
background:rgb(0,0,0,25%);
background:rgb(0,125,165,100%);
border-radius:5px;
min-height:20px;
}
QScrollBar::handle:vertical:hover{
width:10px;
background:rgb(0,125,165,80%);
border-radius:5px;
min-height:20;
}

QScrollBar:horizontal {
height:20px;
background:rgb(0,0,0,0%);
margin:0px,0px,0px,0px;
padding-left:25px;   
padding-right:25px; 
}
QScrollBar::handle:horizontal{
height:10px;
background:rgb(0,125,165,100%);
border-radius:5px;
min-width:20px;
}
QScrollBar::handle:horizontal:hover{
height:10px;
background:rgb(0,125,165,80%);
border-radius:5px;
min-width:20;
}
QToolTip{
font:9pt &quot;楷体&quot;;
color:rgb(0, 0, 0);
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>900</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 125, 165);</string>
      </property>
      <property name="text">
       <string>数据库管理工具</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>560</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>900</width>
        <height>560</height>
       </size>
      </property>
      <widget class="QWidget" name="pageFunction">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="tBtnDbEdit">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="text">
           <string>编辑数据库</string>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/Image/dataSearch.png</normaloff>:/Image/dataSearch.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tBtnDbSend">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="text">
           <string>传送数据库</string>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/Image/offLineInfo.png</normaloff>:/Image/offLineInfo.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageDbEdit"/>
      <widget class="QWidget" name="pageDbSend"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
